File: Python/program24.py
def main():
    
    iValue = int(input("Enter number \n"))
    
    bRet = Check_Divisible(iValue)
    
    if(bRet == True):
        print("Number is divisble by 3 and  5 ")
    else:
        print("Number is not divisble by 3 and  5 ")
    
def Check_Divisible(iNo):
    bflag = False
    if((iNo % 5) == 0) and ((iNo % 3)== 0):
        bflag = True
    else:
        bflag = False
    return bflag
    

    
    
if __name__ == "__main__":
    main()

File: C++/program24.cpp

#include<iostream>
using namespace std;

#define INVALID_ERR -1

unsigned long Factorial(unsigned int iNo)
{
    if(iNo < 0)
    {
        return INVALID_ERR;
    }
    int iCnt = 0;
    unsigned long iFact = 1;

    for(iCnt = 1; iCnt <= iNo; iCnt++)
    {
        iFact = iFact * iCnt;   
       
    }

    return iFact;
  
}

int main()
{
    unsigned int iValue = 0; 
    unsigned long int iRet = 0;

    cout<<"Enter number to get factorial of that number"<<'\n';
    cin>>iValue;

    iRet = Factorial(iValue);

    if(iRet == INVALID_ERR)
    {
        cout<<"Invalid Input"<<'\n';
    }
    else
    {
        cout<<"Factorial is : "<<iRet<<'\n';
    }


    return 0;
}  